beacon-chain/sync/initial-sync/round_robin_test.go

```diff
@@ -87,29 +87,29 @@ func TestRoundRobinSync(t *testing.T) {
 		},
 		{
 			name:               "Multiple peers with failures",
-			currentSlot:        320, // 5 epochs
-			expectedBlockSlots: makeSequence(1, 320),
+			currentSlot:        160, // 10 epochs
```

Comment:
Can you leave the number of slots the same, but just change the epochs? It looks like you reduced the blocks by half and also the finalized epoch by half in the peer data

Reply:
Np. Since epoch duration is 32 slots. I'm going to use 320 blocks and just increased epoch to 8 (double from before)